sassc: invalid option -- 'a'

I'm getting this error when running the script:

> patching file gtk-communitheme/Communitheme/gtk-3.0/_colors.scss
> patching file gtk-communitheme/Communitheme/gtk-2.0/gtkrc
> patching file gtk-communitheme/Communitheme/index.theme
> patching file gtk-communitheme/Communitheme/meson.build
> patching file gtk-communitheme/Communitheme/gtk-3.0/_ubuntu-colors.scss
> The Meson build system
> Version: 0.42.1
> Source dir: /home/norike/Descargas/Communitheme-MATE-master/gtk-communitheme
> Build dir: /home/norike/Descargas/Communitheme-MATE-master/gtk-communitheme/builddir
> Build type: native build
> Project name: gtk-communitheme
> Build machine cpu family: x86_64
> Build machine cpu: x86_64
> Program sassc found: YES (/usr/local/bin/sassc)
> Build targets in project: 2
> ninja: Entering directory `builddir'
> [1/3] Generating generate_gtk with a custom command.
> FAILED: Communitheme/gtk-3.0/gtk.css 
> /usr/local/bin/sassc -a ../Communitheme/gtk-3.0/gtk.scss Communitheme/gtk-3.0/gtk.css
> /usr/local/bin/sassc: invalid option -- 'a'
> See '/usr/local/bin/sassc -h'
> [2/3] Generating generate_gtk-dark with a custom command.
> FAILED: Communitheme/gtk-3.0/gtk-dark.css 
> /usr/local/bin/sassc -a ../Communitheme/gtk-3.0/gtk-dark.scss Communitheme/gtk-3.0/gtk-dark.css
> /usr/local/bin/sassc: invalid option -- 'a'
> See '/usr/local/bin/sassc -h'
> ninja: build stopped: subcommand failed.
> ln: fallo al crear el enlace simbólico '/home/norike/.themes/Communitheme-MATE': El archivo ya existe

Am I overlooking anything?

--- a/install.sh
+++ b/install.sh
@@ -8,8 +8,8 @@ apt install git build-essential meson
 
 printf '\nDownloading, Building, and Installing SassC and LibSass:'
 cd /usr/local/lib/
-git clone https://github.com/sass/sassc.git --branch 3.2.1 --depth 1
-git clone https://github.com/sass/libsass.git --branch 3.2.1 --depth 1
+git clone https://github.com/sass/sassc.git --branch 3.5.0 --depth 1
+git clone https://github.com/sass/libsass.git --branch 3.5.0 --depth 1
 
 cd /usr/local/lib/sassc/
 export SASS_LIBSASS_PATH="/usr/local/lib/libsass"

--- a/install.sh
+++ b/install.sh
@@ -3,7 +3,8 @@
 CURRENTFOLDER=${PWD}
 
 printf '\nInstalling Dependencies:'
-apt install git build-essential
+apt update
+apt install git build-essential meson
 
 printf '\nDownloading, Building, and Installing SassC and LibSass:'
 cd /usr/local/lib/

